.gitlab-ci.yml

```diff
@@ -317,6 +326,7 @@ publish:release-timber:
   tags: [ "runner:main" ]
   only:
     - tags
+    - develop
```

Comment:
if you add this here, doesn't this mean that this task will always be performed when develop builds ? That means we are going to push snapshots every time we merge something in develop, is that correct ?

Reply:
Yes that's the goal